src/js/components/form.js

```diff
@@ -0,0 +1,64 @@
+export class Form {
+
+    API_LINK = "https://api.github.com/users/";
+    REPO_LINK = "/repos";
+    $userNameElement = document.getElementById('userName');
+    $userRepositoryListElement = document.getElementById('repoList');
+    $userAvatarElement = document.getElementById('userAvatar');
+
+    constructor() {
+        this.form = document.getElementById("main-form")
+        this.input = document.querySelector('input')
+        this.submittButton = document.querySelector('button')
+        this.resetButton = document.getElementById('reset');
+    }
+
+    async handleSearchButton() {
```

Comment:
cała ta metoda jest mocno spuchnięta, śmiało można to rozbić na kilka metod np.

handleSearchButton()
fetchData()
renderData()